cli/preprocessor/test.go

```diff
@@ -0,0 +1,68 @@
+package preprocessor
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/goccy/go-yaml"
+	"github.com/kubeshop/tracetest/cli/openapi"
+	"github.com/kubeshop/tracetest/cli/pkg/fileutil"
+	"go.uber.org/zap"
+)
+
+type test struct {
+	logger *zap.Logger
+}
+
+func Test(logger *zap.Logger) test {
+	return test{
+		logger: logger,
+	}
+}
+
+func (t test) Preprocess(ctx context.Context, input fileutil.File) (fileutil.File, error) {
+	var test openapi.TestResource
```

Comment:
this is the same code moved from the `test_run_action.go` file, with a few variables renamed